connectivity/conn_http.h

```diff
@@ -23,31 +23,121 @@ extern "C" {
 #include "mbedtls/net.h"
 #include "mbedtls/ssl.h"
 
-#define HTTP_OK 200
-
+/**< Global pointer points to http resource body */
```

Comment:
"Global pointer points to http resource body" What?
Improve it.

Reply:
I know this is the bad coding style. This global variable is existing for a long time. I will just update the doxygen comment. And mark this for a new issue.

Reply:
Mark at #62 .